openshift/kustomize/tekton/base/tasks/git-conditions.yaml

```diff
@@ -72,7 +72,7 @@ spec:
           buildVar="${1/\-/\_}"
           # convert to uppercase
           buildVar="${buildVar^^}"
-          BUILD=BUILD_${buildVar}=true
+          BUILD_${buildVar}=true
```

Comment:
I believe this is why the `master` branch pipeline isn't working.  However, this may break the `dev` pipeline.  Will need to test the next merge.